src/components/notification/notification.jsx

```diff
@@ -0,0 +1,63 @@
+import styled from "styled-components";
+import { Avatar } from "../avatar/avatar";
+
+/**
+ * Notification Component
+ * @author [J. Hartsek](https://github.com/JHartsek)
+ * @param src the avatar image link
+ * @param alt the avatar image description
+ * @param title the notificaiton origin
+ * @param summary the notfication details
+ * @param status expects 'read' or 'unread'
+ */
+
+const NotificationWrapper = ({ children }) => ({ children });
+
+NotificationWrapper.Container = styled.div`
+  column-gap: 8px;
+  padding: 12px 0 16px 12px;
+`;
+
+NotificationWrapper.Info = styled.div`
+  row-gap: 4px;
```

Comment:
At the moment I think this isn't being applied because this element isn't a flex-box